src/lib/forms-schema/elements/HeadingElement.ts

```diff
@@ -10,4 +10,4 @@ export default Joi.object({
     .required()
     .label('Form Element - Heading Size')
     .valid([1, 2, 3, 4, 5]),
-})
```

Comment:
Why was this added here and not to the others?

Reply:
Because there was a test using this element, when I was trying to figure out where this needed to go. Forgot to remove